recOrder/plugin/main_widget.py

```diff
@@ -955,7 +965,9 @@ def connect_to_mm(self):
                     self.ui.cb_acq_channel.addItem(ch)
                     self.bf_channel_found = True
 
+        print("Checked configs")
         if not config_group_found:
+            print("Config group not found")
```

Comment:
This message seems to be included in the error message:
https://github.com/mehta-lab/recOrder/blob/7e10ae59a1ed28cf28052b97404a2c0eba4369ff/recOrder/plugin/main_widget.py#L971-L974

Reply:
print statements are all because I saw no other (easy) way to get debug output.  What is the logging mechanism used in recOrder?  I saw that napari has something to output plugin logs, but nothing appeared there.  I would greatly encourage some kind of logging to facilitate future trouble shooting.

Reply:
We do have logging with `logging.debug()` etc. in `recOrder`. Unfortunately we do also inherit print statements, and this has been a problem since we are developing a multi-threaded GUI app. See https://github.com/mehta-lab/recOrder/issues/172 for more context and our plan to unify logging with the napari notification system.